artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AmqpCoreConverter.java

```diff
@@ -90,13 +91,16 @@
 
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.PooledByteBufAllocator;
+import org.jboss.logging.Logger;
 
 /**
  *  This class was created just to separate concerns on AMQPConverter.
  *  For better organization of the code.
  * */
 public class AmqpCoreConverter {
 
+   private static final Logger log = Logger.getLogger(AmqpCoreConverter.class);
```

Comment:
I dont see this being used?

Reply:
I was going to use, then removed. you're right

Reply:
resolving it